From acdf605a1b44099b011f323ddd76c44bf4bc2260 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Wed, 7 Sep 2016 14:57:10 +0100
Subject: [PATCH 3/3] Disable building of unused yelp documentation

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 9706d60..7cd42ab 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,7 +2,7 @@ SUBDIRS = common liblightdm-gobject liblightdm-qt
 if COMPILE_TESTS
 SUBDIRS += tests
 endif
-SUBDIRS += data po src doc help
+SUBDIRS += data po src doc
 
 DISTCHECK_CONFIGURE_FLAGS = --enable-introspection --enable-gtk-doc
 
-- 
2.10.0

